crates/sui-protocol-config/src/lib.rs

```diff
@@ -1178,6 +1178,9 @@ impl ProtocolConfig {
                 cfg.feature_flags.narwhal_versioned_metadata = true;
                 if chain != Chain::Mainnet {
                     cfg.feature_flags.commit_root_state_digest = true;
+                }
+
+                if chain != Chain::Mainnet && chain != Chain::Testnet {
```

Comment:
Why do we prefer != and not == DevNet?

Reply:
There is no `Chain::Devnet` defined, since `Chain` is linked to the `ChainIdentifier`, which is based on the genesis checkpoint digest, therefore this will change each time devnet is wiped.